wandb/integration/langchain/wandb_tracer.py

```diff
@@ -285,7 +287,7 @@ def _add_child_run(
     ) -> None:
         """Add child run to a chain run or tool run."""
         try:
-            parent_run.child_runs.append(child_run)
+            parent_run.child_chain_runs.append(child_run)
```

Comment:
why not use `child_run` and change the `parent_run` and `child_run` to `Run` as defined here? https://github.com/hwchase17/langchain/blame/7e967aa4d581bec8b29e9ea44267505b0bad18b9/langchain/callbacks/tracers/schemas.py#L82

Reply:
This child_runs are a part of the `Run` object in langchain that is experimentally being used from `LangchainTracerV2`. We can change this to the `Run` once they stabilize.